netutils/telnetd/telnetd_daemon.c

```diff
@@ -316,6 +307,33 @@ static int telnetd_daemon(int argc, FAR char *argv[])
           close(drvrfd);
         }
 
+#ifdef CONFIG_NETUTILS_TELNETD_USE_POSIX_SPAWNP
+      posix_spawn_file_actions_t file_actions;
+      posix_spawnattr_t attr;
+      FAR char *argv1[2];
```

Comment:
Local variables should be defined at the beginning of the scope (C89 compatibility).

Reply:
done.